### ISSUE TITLE
examples/Atlas/test/testRunningPlanner segfaults

### ISSUE THREAD

**Issue by RussTedrake** [RobotLocomotion/drake#281]
It looks like it's after the planner finishes.  could be connected to the runAtlasWalkingTestMex segfault, too?  I saw this on drake001:

```
Calling addpath_spotless
 Added the lcm jar to your javaclasspath (found via cmake)
[Warning: Currently all of your LCM traffic will be broadcast to the network,
because:
  Route to multicast channel does not run through the loopback interface.
  Consider running: sudo
  /private/tmp/drake-master-22461/drake/util/setup_loopback_multicast.sh
] 
[> In checkDependency at 65
  In addpath_drake at 110
  In addpath_drake at 4] 
 Added the lcmgl jar to your javaclasspath (found via cmake)
 Calling addpath_snopt
Elapsed time is 212.183068 seconds.

------------------------------------------------------------------------
       Segmentation violation detected at Sat Aug  9 20:15:53 2014
------------------------------------------------------------------------

Configuration:
  Crash Decoding     : Disabled
  Current Visual     : None
  Default Encoding   : ISO-8859-1
  MATLAB Architecture: maci64
  MATLAB Root        : /Applications/MATLAB_R2014a.app
  MATLAB Version     : 8.3.0.532 (R2014a)
  Operating System   : Darwin 13.3.0 Darwin Kernel Version 13.3.0: Tue Jun  3 21:27:35 PDT 2014; root:xnu-2422.110.17~1/RELEASE_X86_64 x86_64
  Processor ID       : x86 Family 6 Model 23 Stepping 10, GenuineIntel
  Virtual Machine    : Java 1.7.0_11-b21 with Oracle Corporation Java HotSpot(TM) 64-Bit Server VM mixed mode
  Window System      : None

Fault Count: 1


Abnormal termination:
Segmentation violation

Register State (from fault):
  RAX = 0000000110d5dd70  RBX = 0000000110d5dcb0
  RCX = 0000000000000000  RDX = 0000000136c2a1d0
  RSP = 0000000111a2ce90  RBP = 0000000111a256b8
  RSI = 0000000110d5d5e0  RDI = 0000000110d5dd70

   R8 = 0000000110d5dd70   R9 = 0000000110d5d600
  R10 = 0000000111a2ceba  R11 = 0000000111a2ce90
  R12 = 0000000000000000  R13 = 0000000110d5d630
  R14 = 000000010a3ae0b5  R15 = 0000000000000000

  RIP = 0000127f00000000  RFL = 0000000000000001

   CS = 00007fb90c94eb00   FS = 0000000110d5d710   GS = 000000010a3aa101

Stack Trace (from fault):
[  0] 0x00000001069a7874 /Applications/MATLAB_R2014a.app/bin/maci64/libmwfl.dylib+00030836 _ZN2fl4diag15stacktrace_base7captureERKNS0_14thread_contextEm+00000052
[  1] 0x00000001069ab2aa /Applications/MATLAB_R2014a.app/bin/maci64/libmwfl.dylib+00045738 _ZN2fl4test17terminate_handledEv+00000906
[  2] 0x00000001069aacf5 /Applications/MATLAB_R2014a.app/bin/maci64/libmwfl.dylib+00044277 _ZN2fl4diag13terminate_logEPKcPK17__darwin_ucontext+00000117
[  3] 0x0000000106513d07 /Applications/MATLAB_R2014a.app/bin/maci64/libmwmcr.dylib+00269575 _Z32mnRunPathDependentInitializationv+00003127
[  4] 0x0000000106513ff8 /Applications/MATLAB_R2014a.app/bin/maci64/libmwmcr.dylib+00270328 _Z32mnRunPathDependentInitializationv+00003880
[  5] 0x000000010651181a /Applications/MATLAB_R2014a.app/bin/maci64/libmwmcr.dylib+00260122 mnFatalSignalHandler+00000298
[  6] 0x00007fff886015aa           /usr/lib/system/libsystem_platform.dylib+00013738 _sigtramp+00000026
[  7] 0x00007fb90869de00                                   <unknown-module>+00000000
[  8] 0x000000010a3aa101 /Applications/MATLAB_R2014a.app/bin/maci64/libmex.dylib+00041217 _ZN7Mfh_mex30runMexFileWithSignalProtectionEiPP11mxArray_tagiS2_+00000113
[  9] 0x000000010a3aa2e0 /Applications/MATLAB_R2014a.app/bin/maci64/libmex.dylib+00041696 _ZN7Mfh_mex13dispatch_fileEiPP11mxArray_tagiS2_+00000288
[ 10] 0x000000010a422a9a /Applications/MATLAB_R2014a.app/bin/maci64/libmwm_dispatcher.dylib+00334490 _ZN8Mfh_file11dispatch_fhEiPP11mxArray_tagiS2_+00000602
[ 11] 0x000000010aab2de3 /Applications/MATLAB_R2014a.app/bin/maci64/libmwm_interpreter.dylib+00855523 _ZN12InterpBridge7CallFcnEiPP11mxArray_tagiS2_PKc+00000275
[ 12] 0x000000010aa217cb /Applications/MATLAB_R2014a.app/bin/maci64/libmwm_interpreter.dylib+00260043 inFullFevalFcn+00000283
```

**Comment by RussTedrake** [comment]
fwiw - this is still haunting the build servers:
http://kobol.csail.mit.edu/cd/testSummary.php?project=1&name=examples/Atlas/test/testRunningPlanner&date=2015-04-20

**Comment by RussTedrake** [comment]
useful for testing:
```
matlab -nodisplay -r "addpath_drake; global g_disable_visualizers; g_disable_visualizers=true; cd examples/Atlas/test; testRunningPlanner; exit"
```

**Comment by avalenzu** [comment]
OK. I'll take a look today

**Comment by RussTedrake** [comment]
if you have a reliable way to reproduce it, please add it to the bug.

**Comment by RussTedrake** [comment]
looks like it is not resolved by #1050.

**Comment by RussTedrake** [comment]
won’t fix
